src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
 // write your code here
+// Отримуємо всі елементи <span> з класом "population"
+const populationSpans = document.querySelectorAll('span.population');
+// Масив для збереження числових значень
+const populationValues = [];
+
+// Проходимо через кожен елемент, перевіряємо та конвертуємо текст у число
+populationSpans.forEach((span) => {
+  const populationText = span.textContent.replace(/,/g, '');
+  const populationNumber = parseFloat(populationText);
+
+  // Перевіряємо, чи є значення числом
+  if (!isNaN(populationNumber)) {
+    populationValues.push(populationNumber); // Додаємо в масив, якщо це число
+  }
+});
+
+// Обчислюємо загальну кількість
+const totalPopulation = populationValues.reduce((sum, value) => sum + value, 0);
```

Comment:
You are using the reduce method to calculate the sum of an array of numbers, which is a good use of this method. However, it's important to note that this method will return NaN if the array is empty. You might want to add a check to ensure that the array is not empty before using reduce.